gaozhewei/main.h

```diff
@@ -0,0 +1,14 @@
+//
+// Created by stevegao on 22-6-14.
+//
+
+#ifndef ROS_LEARN_MAIN_H
+#define ROS_LEARN_MAIN_H
+
+
+class main {
+
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Redundant blank line at the end of a code block should be deleted.  [whitespace/blank_line] [3]